packages/web-components/src/components/filter-panel/__stories__/filter-panel.stories.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+import { text, select } from '@storybook/addon-knobs';
+import '../filter-panel';
+import '../filter-panel-heading';
+import '../filter-panel-modal';
+import '../input_select';
+import '../input_select_item';
+import '../filter-group';
+import '../filter-panel-composite';
+import '../checkbox';
+import '../filter-group-item';
+import 'carbon-web-components/es/components/accordion/accordion';
```

Comment:
Is accordion necessary for the component to function? Should this be in `index.ts` instead?